README.md

```diff
@@ -25,11 +27,13 @@ We use Travis CI for testing - add a test if it makes sense, and make sure the b
 
 ## Deploy
 
-These docs are hosted on GitHub Pages. We do a couple things to make this nice:
-
-* Use the [gh-pages](https://github.com/tschaub/gh-pages) library for deploying
-* Use the [404 redirection trick](https://github.com/rafrex/spa-github-pages) to pass path/hash as query param to `/index.html`
+These docs are hosted on GitHub Pages. To deploy what you have locally:
```

Comment:
Can we add the branch - just to make it really clear because [gh-pages](https://github.com/tschaub/gh-pages) refers to the `gh-pages` branch, e.g.:

```
These docs are hosted on GitHub Pages. 

To deploy changes, merge a feature branch into to the master branch and deploy the master branch with the following command:
```